portfolio/target/portfolio-1/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
Looks like there's still some stuff in `target`. Maybe drop these from the commit? I'm actually surprised they aren't in the `.gitignore`

Reply:
It seems like as I write in the portfolio/main/src/index.html it automatically updates /target/portfolio-1/index.html without me writing in it. I'm not sure why this is happening. Any suggestions as to what it could be? I just noticed it today. It seems to update after I run mvn appengine:devserver